src/Phinx/Db/Adapter/AbstractAdapter.php

```diff
@@ -258,4 +263,57 @@ public function isDryRunEnabled()
 
         return ($input && $input->hasOption('dry-run')) ? (bool)$input->getOption('dry-run') : false;
     }
+
+    /**
+     * Adds user-created tables (e.g. not phinxlog) to a cached list
+     *
+     * @param string $tableName The name of the table
+     * @return void
+     */
+    public function addCreatedTable($tableName)
```

Comment:
I think all these new methods should be protected

Reply:
Done